node/manifest.ts

```diff
@@ -106,6 +100,7 @@ const generateManifest = ({
       name: declaration.name,
       pattern: serializePattern(pattern),
       generator: declaration.generator,
+      on_error: declaration.onError ?? functionConfig[func.name]?.onError,
```

Comment:
the order here is important: failure modes can be specified on a declaration level, allowing the same function to have different failure modes on different paths. this takes precedence over settings defined in ISC.

Reply:
Hmm I think this might be too confusing. Can we just make this configurable on a per-function basis?

Reply:
okidoke 👍 i'll move it to the functions config object then

Reply:
done in https://github.com/netlify/edge-bundler/pull/328/commits/a704875145029f00746734c7eac9c22a72c13c83